### DISCUSSION TITLE
Can we change the url to nr-fds-theme? Or is that crazy?

### DISCUSSION THREAD

**Discussion by DerekRoberts** [bcgov/nr-theme#65] [Ideas]
Hey @jackwingsam!  Let's mess around with GitHub Discussions using your issue as a starting point.  :)

TLDR; sure!

https://github.com/bcgov/nr-fsa-theme/issues/64

**Comment by jackwingsam** [comment]
Neat! The change from FSA to FDS is imminent I hear.

**Reply by DerekRoberts** [reply]
Then we do it!  I just think we need to loop in with @MCatherine1994 to make sure this doesn't break npmjs.com publishing.
